apps/bookmark/js/bookmark_editor.js

```diff
@@ -86,30 +86,29 @@ var BookmarkEditor = {
     this.appIcon.addEventListener('load', this.appIconListener);
     this.appIcon.setAttribute('src', iconURL);
   },
-  
+
   _handleAppIconLoad: function _handleAppIconLoad() {
     this.appIconPlaceholder.classList.add('hidden');
     this.appIcon.classList.remove('hidden');
     this.appIcon.removeEventListener('load', this.appIconListener);
   },
-  
+
   _fetchManifest: function bookmarkEditor_fetchManifest(manifestURL) {
     var manifestPromise = window.WebManifestHelper.getManifest(manifestURL);
-    
+
     manifestPromise.then((function(manifestData) {
       if (manifestData) {
         this.installAppButtonListener = this._installApp.bind(this);
         this.installAppButton.addEventListener('click',
           this.installAppButtonListener);
-        this.appNameText.textContent = manifestData.short_name ||
-          manifestData.name;
+        this.appNameText.textContent = manifestData.displayName;
```

Comment:
This won't work, manifestData comes from WebManifestHelper which is a different thing to ManifestHelper, it fetches and parses W3C web app manifests for apps which are not installed.